app/views/content/landing/class-of-now/_header.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= render Content::CampaignHeroComponent.new(
+  title: ["Join The", "Class of Now"],
```

Comment:
@philspray This wouldn't normally be our GIT style, to capitalise in headings but I appreciate that this a campaign

@EllieNodder any thoughts on this? Should we at least have a lowercase 't' on 'The'?